### PR TITLE
Add selectedType function and unit tests

### PR DESCRIPTION
**Pull request by clewis1**
https://companieshouse.atlassian.net/browse/IDVA3-1022

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/routers/pscTypeRouter.ts [hunk 1]
```diff
@@ -20,9 +20,18 @@ router.get(Urls.PSC_TYPE, authenticate, handleExceptions(async (req: Request, re
...
-    const nextPageUrl = getUrlWithTransactionIdAndSubmissionId(PrefixedUrls.INDIVIDUAL_PSC_LIST, req.params.transactionId, req.params.submissionId);
+    const nextPageUrl = getUrlWithTransactionIdAndSubmissionId(selectPscType(selectedType), req.params.transactionId, req.params.submissionId);
     res.redirect(addSearchParams(nextPageUrl, { lang }));
 }));
 
+const selectPscType = (pscType: any): string => {
+    switch (pscType) {
+    case "individual": return PrefixedUrls.INDIVIDUAL_PSC_LIST;
+    case "rle": return PrefixedUrls.RLE_LIST;
+    default: return PrefixedUrls.INDIVIDUAL_PSC_LIST;
```
Comment (hepsimo) [inline src/routers/pscTypeRouter.ts:33]: The default case (neither option selected) will get error handling later, so best add a TODO comment as reminder.

**Review by hepsimo — CHANGES_REQUESTED** [review]
Changes look good, just need a TODO comment for a reminder

**Review by hepsimo — CHANGES_REQUESTED** [review]
Changes look good, just need a TODO comment added

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by hepsimo — APPROVED** [review]
Changes look good

**Pull request merged**